src/chainparams.h

```diff
@@ -78,6 +78,17 @@ class CChainParams
     const CCheckpointData& Checkpoints() const { return checkpointData; }
     const ChainTxData& TxData() const { return chainTxData; }
     void UpdateVersionBitsParameters(Consensus::DeploymentPos d, int64_t nStartTime, int64_t nTimeout);
+    int SwitchAllium(int nHeight) const {
```

Comment:
For readability sake, function could be reduced down to:

`return strNetworkID == CBaseChainParams::TESTNET && nHeight > 10`

Reply:
I'd rather suggest changing this to a "GetPoWAlgo" with fall through cases. Just for future-proofing. Also, rather than returning int's, you could define a global int-based enum. Would make it easier to read any comparisons in the rest of the code.